ActiasRuntimeTools/main.cpp

```diff
@@ -38,66 +91,55 @@ int main(int argc, char** argv)
         arguments, "version", "Show the version of Actias runtime and SDK", { 'v', "version" }, args::Options::KickOut);
 
     args::Group commands(parser, "SDK Commands");
-    const args::Command build(commands, "build", "Build an Actias project", [](args::Subparser& p) {
+    const args::Command build(commands, "build", "Build an Actias project", [&result](args::Subparser& p) {
         args::Positional<std::string> executable(
             p, "executable", "The native OS executable to build (dll, exe, so, etc.)", args::Options::Required);
         p.Parse();
 
-        LibraryLoader loader("ActiasSDK", true);
+        build(executable.Get().data(), result);
+    });
 
-        String path;
-        path.Append(executable.Get().data(), executable.Get().size());
+    const args::Command run(commands, "run", "Run an ACBX executable file", [&result](args::Subparser& p) {
+        args::Positional<std::string> executable(p, "executable", "The executable (ACBX) file to run", args::Options::Required);
+        p.Parse();
 
-        auto ExecutableRead = File::ReadAllBytes(path);
+        String executableName = executable.Get().data();
+        StringSlice executableExtension(((StringSlice)executableName).FindLastOf('.'), ((StringSlice)executableName).end());
 
-        if (!ExecutableRead)
+        if (executableExtension == ".acbx")
         {
-            std::cout << "Executable file not found" << std::endl;
-            return;
+            LibraryLoader loader(executableName, false);
```

Comment:
Давай при объявлении переменных таким образом будем использовать фигурные скобки:
```suggestion
            LibraryLoader loader{executableName, false};
```
Иначе, если так вызвать дефолтный конструктор какого-нибудь класса, компилятор посчитает это за объявление функции, например:
```cpp
struct Foo
{
    Foo() { /* ... */ }
};

Foo foo(); // <-- объявилась локальная функция, возвращающая Foo
Foo foo{}; // <-- локальная переменная типа Foo, инициализированная
           //     дефолтным конструктором
```